core_lib/src/canvaspainter.cpp

```diff
@@ -131,7 +134,7 @@ void CanvasPainter::renderPreLayers(QPixmap *pixmap)
 
 void CanvasPainter::renderPreLayers(QPainter& painter)
 {
-    if (mOptions.nShowAllLayers > 0)
+    if (mOptions.elayerVisibility != CanvasPainterOptions::HIDDEN)
```

Comment:
I personally think that goes against the UI expected behaviour, even if I agree that a solo camera layer is useless. I will make the change.

Reply:
It definitely goes against the expected behavior of the layer soloing, but I've always pushed the idea that when you're on the camera layer, you should see exactly what the camera sees and exactly what it will look like when exporting. So what we've got is a conflict between these two expected behaviors.